dev_tools/spec/lib/solidus/release_drafter/builder_spec.rb

```diff
@@ -117,35 +120,35 @@
       end
 
       it 'prepends given text' do
-        builder = described_class.new(draft: draft, categories: categories, prepend: "## List of changes\n\n")
+        builder = described_class.new(draft: draft, categories: categories, prepend: "## List of changes\n\n", number_link_builder: number_link_builder, user_link_builder: user_link_builder)
 
         expect(
           builder.add(title: 'New entry', number: 10, user: 'alice', categories: [core, api]).content
         ).to eq <<~TXT.chomp
           ## List of changes
 
           ## Solidus Core
-          - New entry #10 (@alice)
+          #{entry.('New entry', 10, 'alice')}
```

Comment:
nit: here and elsewhere I think would be better to have the final entry spelled, even if it clearly means having some more duplication/repetition. It happened to me multiple times that  this kind of helpers hide additional bugs that are hard to catch before hitting production.

Reply:
I agree entirely, @elia, and that's my way to go 99% of the time. However, IMHO, it'd be too much in this case as we've added the two markdown links to the entry. The tradeoff of understanding the format in a single place is more important for me in this case.